apps/bridge-dapp/src/types/index.ts

```diff
@@ -53,3 +52,29 @@ export interface SocialConfigsType extends Omit<ExternalLink, 'name'> {
    */
   Icon: (props: IconBase) => JSX.Element;
 }
+type ColorIndex =
+  | '0'
+  | '10'
+  | '20'
+  | '30'
+  | '40'
+  | '50'
+  | '60'
+  | '70'
+  | '80'
+  | '90'
+  | '100'
+  | '110'
+  | '120'
+  | 'DEFAULT';
+/**
+ * The Webb color type
+ */
+export interface WebbColorsType {
+  mono: Record<ColorIndex, string>;
+  purple: Record<ColorIndex, string>;
+  blue: Record<ColorIndex, string>;
+  green: Record<ColorIndex, string>;
+  yellow: Record<ColorIndex, string>;
+  red: Record<ColorIndex, string>;
+}
```

Comment:
Do we need this? I thought the colour schema was being served from tailwind presets? @AtelyPham can you confirm?

Reply:
This was getting it working ,and this can be removed once @AtelyPham  confirms

Reply:
Removed